pontoon/base/tests/test_models.py

```diff
@@ -0,0 +1,75 @@
+from django_nose.tools import assert_equal
+
+from pontoon.base.models import Translation
+from pontoon.base.tests import (
+    assert_redirects,
+    EntityFactory,
+    LocaleFactory,
+    ProjectFactory,
+    ResourceFactory,
+    TranslationFactory,
+    UserFactory,
+    TestCase
+)
+from pontoon.base.utils import aware_datetime
+
+
+class LocaleTests(TestCase):
+    def test_latest_activity_translated(self):
+        """
+        If latest activity in Translation QuerySet is translation submission,
+        return submission date and user.
+        """
+        user0 = UserFactory.create()
```

Comment:
These users are shared and could go in a `setUp` method. You could also use `UserFactory.create_batch(2)` to create them both at the same time.